deployment/k8sPaiLibrary/maintainlib/add.py

```diff
@@ -93,6 +93,11 @@ def job_executer_add_work_node(self):
             self.logger.error("Failed to update the /etc/hosts on {0}".format(self.node_config['hostip']))
             sys.exit(1)
 
+        commandline = "sudo ./{0}/prerequisites-install.sh {0}".format(self.jobname)
+        if common.ssh_shell_with_password_input_paramiko(self.node_config, commandline) == False:
+            self.logger.error("Failed to install docker-ce on {0}".format(self.node_config['hostip']))
```

Comment:
docker ce?

Reply:
yes, consider this, this is a little like install docker-ce and uninstall docker-ce type problem. If current PAI default not uninstall docker when clean k8s, i will remove this uninstall cmd and let user customized uninstall ntp? What's your suggestion?

Reply:
Please change the log.